lib/daru/view/adapters/highcharts.rb

```diff
@@ -42,10 +40,17 @@ def init(data=[], options={})
             # all the options present in `options` and about the
             # series (means name, type, data) used in f.series(..)
             f.options = options.empty? ? LazyHighCharts::HighChart.new.defaults_options : options
-
-            series_type = options[:type] unless options[:type].nil?
-            series_name = options[:name] unless options[:name].nil?
-            f.series(type: series_type, name: series_name, data: data_new)
+            # For multiple series when data is in a series format as in
+            # HighCharts official examples
+            # TODO: Add support for multiple series when data as Daru::DataFrame/Daru::Vector
+            if data.is_a?(Array) && data[0].is_a?(Hash)
```

Comment:
It is already working for multiple series, since it is checking for array in `guess_data ` , right ?